BuildingSync.xsd

```diff
@@ -17787,4 +17790,468 @@
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta3" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In the 4p models, this is the change point (as there is only one change point).  In the 5p models, this is the lower value change point, which in CalTRACK terms is referred to as the heating change point.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta4" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In the 5p model, this is the upper value change point, which in CalTRACK terms is referred to as the cooling change point.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                            </xs:sequence>
+                          </xs:complexType>
+                        </xs:element>
+                        <xs:element name="TimeOfWeekTemperatureModel">
+                          <xs:complexType>
+                            <xs:sequence>
+                              <xs:element name="DUMMY"/>
+                            </xs:sequence>
+                          </xs:complexType>
+                        </xs:element>
+                      </xs:choice>
+                    </xs:complexType>
+                  </xs:element>
+                  <xs:element name="DerivedModelPerformance">
+                    <xs:annotation>
+                      <xs:documentation>Characterization of the performance of the model.</xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                      <xs:sequence>
+                        <xs:element name="RSquared" minOccurs="0" maxOccurs="1">
+                          <xs:simpleType>
+                            <xs:restriction base="xs:decimal">
+                              <xs:minInclusive value="0"/>
+                              <xs:maxInclusive value="1"/>
+                            </xs:restriction>
+                          </xs:simpleType>
+                        </xs:element>
+                        <xs:element name="AdjustedRSquared" minOccurs="0" maxOccurs="1">
+                          <xs:simpleType>
+                            <xs:restriction base="xs:decimal">
+                              <xs:minInclusive value="0"/>
+                              <xs:maxInclusive value="1"/>
+                            </xs:restriction>
+                          </xs:simpleType>
+                        </xs:element>
+                        <xs:element name="RMSE" minOccurs="0" maxOccurs="1">
```

Comment:
Annotation?

Reply:
The Root Mean Square Error (RMSE) is the standard deviation of the residuals.  It is calculated as follows: RMSE = sqrt( sum((y_i - yhat_i)^2)  / (n - p) ).  Here, y_i is the ith observation of the dependent variable, yhat_i is the regression models predicted value of y_i, n is the number of data points or periods in the baseline period, and p is the number of parameters or terms in the baseline model (ASHRAE Guideline 14-2014).